security/providers/abac/src/main/java/io/helidon/security/providers/abac/AbacProvider.java

```diff
@@ -224,39 +234,53 @@ private void validateCustom(EndpointConfig epConfig, Errors.Collector collector)
     }
 
     private void validateConfig(EndpointConfig config, Errors.Collector collector) {
-        config.config("abac")
-                .ifPresent(abacConfig -> abacConfig.asMap()
-                        .ifPresent(theMap -> {
-                            int attributes = 0;
-                            int unsupported = 0;
-                            List<String> unsupportedKeys = new LinkedList<>();
-
-                            for (String key : theMap.keySet()) {
-                                attributes++;
-                                if (!supportedConfigKeys.contains(key)) {
-                                    unsupported++;
-                                    unsupportedKeys.add(key);
-                                }
-                            }
+        config.config(CONFIG_KEY)
```

Comment:
same as on the other PR... I wonder whether there should be an orElse to warn the developer

Reply:
I am not sure I understand - the ABAC provider does not require any configuration, so there is not much to do here.

Reply:
well, say some developer has the same thought about the empty `abac` key in the `application.yaml`. Normally an empty key - especially when you come from Spring Boot - is having the connotation of a no-op. So, if you remove the `abac` key it would be helpful to get at least on FINE or FINER a warning that tells you that ABAC support is turned off

Reply:
It works slightly differently.
If you add the `abac` key to the configuration, security would pick the security provider up and configure it.
If remove the `abac` key from configuration, security will never try to initialize the ABAC provider.

Reply:
yes. But many people will expect that

```yaml
   abac:
```
would be a no-op.

With Spring the pattern normally is
```yaml
  abac:
    enabled: true
```
which is more explicit